internal/services/dedicatedcloud/dedicatedcloud.go

```diff
@@ -6,23 +6,767 @@ package dedicatedcloud
+			connectionStateIndicator = "🟢"
+		}
+		allFilers[i]["connectionStateIndicator"] = connectionStateIndicator
+	}
+	object["filers"] = allFilers
+
+	// Calculate totals
+	totalCores := 0
+	totalRAM := 0.0
+	totalVMs := 0
+	totalDiskSpace := 0.0
+
+	// Sum from hosts
+	for _, host := range hosts {
+		// Sum cores
+		if cpuNumRaw, ok := host["cpuNum"]; ok && cpuNumRaw != nil {
+			var cpuNumValue float64
+			switch v := cpuNumRaw.(type) {
+			case json.Number:
+				val, err := v.Float64()
+				if err == nil {
+					cpuNumValue = val
+				}
+			case float64:
+				cpuNumValue = v
+			case int:
+				cpuNumValue = float64(v)
+			case int64:
+				cpuNumValue = float64(v)
+			case int32:
+				cpuNumValue = float64(v)
+			}
+			totalCores += int(cpuNumValue)
+		}
+
+		// Sum RAM
+		if ram, ok := host["ram"].(map[string]any); ok {
+			if ramValue, ok := ram["value"].(float64); ok {
+				totalRAM += ramValue
+			} else if ramValueRaw, ok := ram["value"]; ok && ramValueRaw != nil {
+				switch v := ramValueRaw.(type) {
+				case json.Number:
+					val, err := v.Float64()
+					if err == nil {
+						totalRAM += val
+					}
+				case float64:
+					totalRAM += v
+				case int:
+					totalRAM += float64(v)
+				case int64:
+					totalRAM += float64(v)
+				}
+			}
+		}
+
+		// Sum VMs
+		if vmTotalRaw, ok := host["vmTotal"]; ok && vmTotalRaw != nil {
+			switch v := vmTotalRaw.(type) {
+			case json.Number:
+				val, err := v.Int64()
+				if err == nil {
+					totalVMs += int(val)
+				}
+			case float64:
+				totalVMs += int(v)
+			case int:
+				totalVMs += v
+			case int64:
+				totalVMs += int(v)
+			case int32:
+				totalVMs += int(v)
+			}
+		}
+	}
+
+	// Sum disk space from filers
+	for _, filer := range allFilers {
+		if size, ok := filer["size"].(map[string]any); ok {
+			var sizeValue float64
+			if sizeValueRaw, ok := size["value"]; ok && sizeValueRaw != nil {
+				switch v := sizeValueRaw.(type) {
+				case json.Number:
+					val, err := v.Float64()
+					if err == nil {
+						sizeValue = val
+					}
+				case float64:
+					sizeValue = v
+				case int:
+					sizeValue = float64(v)
+				case int64:
+					sizeValue = float64(v)
+				case int32:
+					sizeValue = float64(v)
+				}
+				// Convert to GB if needed
+				if sizeUnit, ok := size["unit"].(string); ok {
+					if strings.ToUpper(sizeUnit) == "TB" {
+						sizeValue *= 1000 // Convert TB to GB
+					} else if strings.ToUpper(sizeUnit) == "MB" {
+						sizeValue /= 1000 // Convert MB to GB
+					}
+				}
+				totalDiskSpace += sizeValue
+			}
+		}
+	}
+
+	// Format totals
+	object["totalCores"] = totalCores
+	object["totalRAM"] = fmt.Sprintf("%.0f GB", totalRAM)
+	object["totalVMs"] = totalVMs
+	object["totalDiskSpace"] = fmt.Sprintf("%.0f GB", totalDiskSpace)
+
+	// Fetch IAM information from parent dedicatedcloud
+	dedicatedcloudEndpoint := fmt.Sprintf("/dedicatedCloud/%s", url.PathEscape(args[0]))
```

Comment:
not sure we should re-fetch this here, it's ok it the user has two commands to do to fetch the IAM information and this command